.rubocop.yml

```diff
@@ -13,8 +13,14 @@ AllCops:
     - 'db/schema.rb'
     - 'script/**/*'
     - 'vendor/bundle/**/*'
+Naming/FileName:
+  Exclude:
```

Comment:
what version of rubocop are you using, this is supposed to be solved in Rubocop 0.49.0 and we are using Rubocop 0.49.1: https://github.com/bbatsov/rubocop/blob/master/CHANGELOG.md#0490-2017-05-24

Reply:
If this is needed it seems to be a bug, and not something we should exclude in our configuration.

Reply:
so what is the fix here?